pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "cluster-operator"
 	source      = "https://github.com/giantswarm/cluster-operator"
-	version     = "5.8.1-dev"
+	version     = "5.8.0"
```

Comment:
we need a patch version (i.e. 5.8.0-patch1) reconciling the same version `5.8.0`